DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -996,6 +996,17 @@ public class DcMsg {
         return dc_msg_is_setupmessage(messagePointer) == 1
     }
 
+    public var hasHtml: Bool {
+        return dc_msg_has_html(messagePointer) == 1
+    }
+
+    public lazy var html: String = {
+        guard let cString = dc_get_msg_html(DcContext.shared.contextPointer, dc_msg_get_id(messagePointer)) else { return "" }
```

Comment:
`dc_get_msg_html()` belongs to the context object, not to the message object; if we move it there, we also do not need the deprecated DcContext.shared.contextPointer.

Reply:
I changed that